EducationalCenter/EducationalCenter.Angular/ClientApp/src/app/shared/models/courseDTO.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface CourseDTO {
```

Comment:
DTO suffix usually not used on front-end. Just Course

Reply:
fixed